### PR TITLE
Fix: slack message filesize

### PR DESCRIPTION
**Pull request by Fraggle**
## Description

Calculate the filesize from the blob size, not just by counting characters.
Improve error message to help future debugging.

## Risk

Low

## Deploy Plan

Deploy `front`, Deploy `connectors`

### CODE REVIEW

File: front/lib/api/files/utils.ts [hunk 1]
```diff
@@ -63,7 +63,8 @@ export const parseUploadRequest = async (
         return new Err({
           name: "dust_error",
           code: "file_too_large",
-          message: "File is too large.",
+          message:
+            "File is too large or the size passed to the File instance in the DB does not match the size of the uploaded file.",
```
Comment (spolu) [inline front/lib/api/files/utils.ts:67]: Hmm no need to mention "DB" here?

**Review by spolu — APPROVED** [review]
LGTM 🙏

File: front/lib/api/files/utils.ts (hunk 1 above)
Comment (Fraggle) [reply front/lib/api/files/utils.ts:67]: It's the one in the DB i'm referring to, how would you call it ?

**Comment by spolu** [comment]
Note we also need to change the toFileBasedContentFragment (name to be verified :-)) in front in the same way 👍

**Comment by Fraggle** [comment]
> Note we also need to change the toFileBasedContentFragment (name to be verified :-)) in front in the same way 👍

In that case we are directly passing a string and thus not parsing any file upload, this error cannot be raised.

File: front/lib/api/files/utils.ts (hunk 1 above)
Comment (spolu) [reply front/lib/api/files/utils.ts:67]: @Fraggle "File instance in the DB" I think makes little sense to a developer (assuming this is developer visible).

Maybe File is too large or there is a mismatch of size between the data received and the size specified at creation?

The real question is why do we even need that field?

Comment (spolu) [reply front/lib/api/files/utils.ts:67]: And what happens when you set it above?

**Pull request merged**